src/components/calculator.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
+import React, { useState } from 'react';
+import Button from '../components/Button.js';
```

Comment:
- [ ] This component produces an error,  please consider changing the name of the component to `Button.js` or changing this line to `import Button from '../components/button.js';` to fix this problem

![Screenshot_20220611_170654](https://user-images.githubusercontent.com/25227022/173195965-b87fcadc-2c4a-482c-989e-6950d90cb0d7.png)